authority-portal-frontend/src/app/shared/components/business/footer-links/footer-links.component.ts

```diff
@@ -19,11 +19,8 @@ import {FooterLink} from './footer-link.model';
   templateUrl: './footer-links.component.html',
 })
 export class FooterLinksComponent {
-  @HostBinding('class.flex')
-  @HostBinding('class.justify-center')
-  @HostBinding('class.items-center')
-  @HostBinding('class.gap-4')
-  cls = true;
+  @Input()
+  isExpandedMenu: boolean = true;
```

Comment:
done

Reply:
I meant the solution with the

```html
<app-footer-links [ngClass]="{'flex-row': isExpandedMenu, 'flex-col max-w-min': !isExpandedMenu}"></app-footer-links>
```

When I said "If you had a reason to have two "modes" of the footer, since you need it in multiple places", I meant that we don't have a reason to have two modes, so we should not incorporate that here. This style of specializing something for something that barely makes sense is something we should try to prevent if we have another easily accessible solution

![image](https://github.com/sovity/authority-portal/assets/30114507/4e82b2e2-efc6-4821-8e22-b31f4e4ccf99)